annotator/src/main/java/org/cbioportal/annotator/internal/GenomeNexusImpl.java

```diff
@@ -596,11 +600,11 @@ public String convertToHgvs(MutationRecord record)
         String start = record.getSTART_POSITION();
         String end = record.getEND_POSITION();
         String ref = record.getREFERENCE_ALLELE();
-        String var = getTumorSeqAllele(record);
+        String var = MafUtil.resolveTumorSeqAllele(record.getREFERENCE_ALLELE(), record.getTUMOR_SEQ_ALLELE1(), record.getTUMOR_SEQ_ALLELE2());
 
         String prefix = "";
         if(ref.equals(var)) {
-            log.warn("Warning: Reference allele extracted from " + chr + ":" + start + "-" + end + " matches alt allele. Sample: " + record.getTUMOR_SAMPLE_BARCODE());
+            LOG.warn("Warning: Reference allele extracted from " + chr + ":" + start + "-" + end + " matches alt allele. Sample: " + record.getTUMOR_SAMPLE_BARCODE());
```

Comment:
shouldn't this be logged as an ERROR instead of WARNING? Should these cases be filtered out when loading in cBioPortal?

Reply:
@pieterlukasse actually wondering how this works with the validator. Does it do any MAF file validation? Cause if it does, it should probably indicate these warnings as well.

Reply:
@inodb good point...I had already asked @dionnezaal to implement a few of these checks just to see how much issues we would find in the current batch of provisional TCGA files.

Reply:
If we make them ERROR in the validation, then they would be flagged by the Circle CI at datahub as well.